### ISSUE TITLE
Incorporate WCAG changes from SSB:s branch in Assembla

### ISSUE THREAD

**Issue by likp** [statisticssweden/PxWeb#50]
SSB have made changes in PxWeb to make it more WCAG complaint. Although these changes are in  our old repo in Assembla.

Components that are effected in PcAxis.Web.Controls  are:
- [x] VariabelSelector
- [x] VariableSelectorValueSelect
- [x] Information
- [x] NavigationFlow
- [x] Footnote
- [x] TableQuery
- [x] SearchValues
- [x] SelectFormGroup
- [x] Table
- [x] Sum
- [x] Pivot
- [x] PerPart
- [x] DeleteVariable
- [x] ChangeValueOrder
- [x] ChangeTexts
- [x] ChangePresentation

Components/Pages that are effected in PxWeb are:
- [ ] Selection*
- [ ] PxWeb.Master*
- [ ] Pcaxis.web.control.js
- [ ] Features-Api-General
- [ ] Features-Api-General.cs*
- [ ] IRouteExtender
- [ ] PxContext*
- [ ] SavedQueryPaxiomCache
- [ ] ApiSettings * (inte logflush)
- [ ] SavedQuerySettings
- [ ] SelectionSettings
- [ ] ScreenViewSerializerAdapter*
- [ ] ErrorGeneral
- [ ] InformationPresentation
- [ ] InformationSelection
- [ ] Presentation.Master*
- [ ] Pcaxis.web.client.js*
- [ ] Pcaxis.web.controls*
- [ ] Admin.css* (take the indentation from SSB and keep the classes from master at Github)
- [ ] designSystem.css* (Clean up stuff that note in use and move it to pxweb.css)
- [ ] All other Css:er that are changed (do a diff between the Assembla branch and the Github branch)
- [ ] SavedQueryHandler*
- [ ] MetadataSystemControl
- [ ] SavedQueryCreate

Item marked with a * indicates that they can not just be taken from Assembla.
There are also a couple of new files e.g images that should be added to this repo. Do a folder diff between the branch in Assembla  vs the master branch on Github.
